app/models/item.rb

```diff
@@ -21,7 +21,8 @@ def subtotal(order_id)
   end
 
   def high_bid
-    bids.maximum('price')
+    high_bid = bids.maximum('price')
+    high_bid ? high_bid : 0.00
```

Comment:
Lets talk to Josh at our next check-in. I've gotten different feedback from different instructors.